src/features/element-hiding.js

```diff
@@ -25,8 +26,9 @@ function collapseDomNode (element, rule, previousElement) {
     }
     const type = rule.type
     const alreadyHidden = hiddenElements.has(element)
-
-    if (alreadyHidden) {
+    const alreadyModified = modifiedElements.has(element) && modifiedElements.get(element) === rule.type
```

Comment:
Don't want to accidentally apply the same modifications to the same element multiple times, but also want to be able to apply a `modify-style` rule and a `modify-attr` rule to the same element if needed.